### ISSUE TITLE
Background does not mutate when setting app:riv_mutate_background="true" layout xml

### ISSUE THREAD

**Issue by jrocharodrigues** [vinc3m1/RoundedImageView#130]
I've cloned the repo and made some changes to the example app to make some tests.

![image](https://cloud.githubusercontent.com/assets/2815253/11032549/4fed1c64-86d4-11e5-9fca-6357ec42adf0.png)

The top image is what i get with this xml:

```
<com.makeramen.roundedimageview.RoundedImageView
        android:id="@+id/imageView1"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_margin="10dp"
        android:background="#FEAB65"
        android:scaleType="center"
        android:src="@drawable/chn_bandrj"
        app:riv_border_color="@color/border"
        app:riv_border_width="3dip"
        app:riv_corner_radius="4dip"
        app:riv_mutate_background="true"
        app:riv_oval="false" />
```

The image and the border is ok, but the background does not have corners rounded.

The bottom one is what i get when i set the background on the activity class using:

```
RoundedImageView mImageView = (RoundedImageView) findViewById(R.id.imageView2);
      mImageView.mutateBackground(true);
      mImageView.setBackgroundColor(Color.parseColor("#FEAB65"));
```

 and with the following xml:

```
<com.makeramen.roundedimageview.RoundedImageView
        android:id="@+id/imageView2"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_margin="10dp"
        android:scaleType="center"
        android:src="@drawable/chn_bandrj"
        app:riv_border_color="@color/border"
        app:riv_border_width="3dip"
        app:riv_corner_radius="4dip"
        app:riv_oval="false" />
```

I expected to get the same results on both images. I'm i doing something wrong?

Thanks in advance

**Comment by vinc3m1** [comment]
Being addressed in #140, and is the same underlying problem as #137

**Comment by vinc3m1** [comment]
should be resolved with #140
